### PR TITLE
feat: Input

### PR DESCRIPTION
**Pull request by Shivanshsharma01**
image
image

### CODE REVIEW

**Review by shivangisharma01 — APPROVED** [review]
The code is clean and easily understandable, looks good to merge.

File: Components/Input/input.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+label {
+    display: block;
+    margin-bottom: 2px;
+}
+
+.input-simple {
+    border: 1px solid #999;
+}
+
+input {
```
Comment (RakshithBhat03) [inline Components/Input/input.css:20]: Don't add styles directly to the element. Use a separate class.

File: Components/Input/input.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    background: var(--color-white);
+    padding: 10px 11px;
+    float: left;
```
Comment (RakshithBhat03) [inline Components/Input/input.css:23]: Not sure about float, but can it be implemented with either flex or other positioning properties.

**Review by RakshithBhat03 — COMMENTED** [review]
Suggested changes. Everything else looks good.

**Pull request merged**